src/reaper_osara.cpp

```diff
@@ -915,15 +934,16 @@ void postInvertTrackPhase(int command) {
 	MediaTrack* track = GetLastTouchedTrack();
 	if (!track)
 		return;
-	outputMessage(isTrackPhaseInverted(track) ?
-		translate("phase inverted") :
-		translate("phase normal"));
+	outputMessage(
+		isTrackPhaseInverted(track) ? translate("phase inverted")
+																: translate("phase normal")
```

Comment:
Definitely something wacky going on with ternary operators. It's like it's trying to align horizontally instead of using continuation indentation.